### ISSUE TITLE
"nouveautés" don't show up in FF Ubuntu

### ISSUE THREAD

**Issue by scoid** [HUB-HUmanBeing/SystemD#267]
=
![Firefox_Screenshot_2020-02-02T09-48-44 218Z](https://user-images.githubusercontent.com/2099251/73606361-536e2500-45a1-11ea-98e6-18657089e5e0.png)

**Comment by ioqt00** [comment]
Which version do you use ?

**Comment by scoid** [comment]
@ioqt00 72.0.2 (64-Bit)

**Comment by scoid** [comment]
I have to admit, that when I switched alll trackers on, I can see more in french, not italian.
Can we get rid step by step of more trackers?

![Firefox_Screenshot_2020-02-02T13-22-03 646Z](https://user-images.githubusercontent.com/2099251/73608805-30eb0480-45bf-11ea-8ed3-562c1bfac342.png)

**Comment by ioqt00** [comment]
Yes I'm working on internationalization of newsfeed, at this time it's only french... What do you mean by trackers, do you use the FF tools or another extension ? Do you have a report on the blocked trackers ?

**Comment by ioqt00** [comment]
Ok I think I can see what the problem is. Thanks ! 
If you can link the blocker report so I can verify, it could be nice :) 
Mine doesn't block links to imgur hosting service, may be a stricter one does... 

@robinBanquo Une solution alternative à Imgur ?

**Comment by scoid** [comment]
imgur.com
i.imgur.com
system-d.eu.meteorapp.com

**Comment by robinBanquo** [comment]
@ioqt00 I dont know anyting as easy as imgur, If you foind something, no problem

**Comment by robinBanquo** [comment]
you can use the public folder but you will have to deploy
